lua/telekasten.lua

```diff
@@ -243,7 +304,8 @@ local function create_note_from_template(
     title,
     filepath,
     templatefn,
-    calendar_info
+    calendar_info,
+    relative_dates
```

Comment:
The parameter `relative_dates` here is misleading. In the whole codebase, `create_note_from_template` is never called with this parameter. Hence, it's always `nil` - and gets passed as `nil` to to `linesubst()`. I suggest removing the parameter from here. Will elaborate on how this fits into the whole - further down the review.